vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -127,9 +130,9 @@ void AdsImpl::InitializeStep4(const Result result) {
 
   NotificationAllowedCheck(false);
 
-  if (IsMobile()) {
-    CleanPostedAsRegistryAfterReboot();
-  }
+#if defined(OS_ANDROID)
```

Comment:
Please see my comments as we should be using IsMobile, IsAndroid and not #ifdef

Reply:
Also this is not necessarily after just a reboot so we should be using same naming connect rations elsewhere in ads of RemoveNotificationsIfNeeded. See review comments I have been posting

Reply:
I'm not familiar with this naming convention. Is it documented? Where can I find it?

Reply:
This is the naming convention we have adopted in ads service and ads lib so we need to be consistent.

Reply:
don't agree